next.config.js

```diff
@@ -15,7 +15,7 @@ const securityHeaders = [
       };` +
       `style-src 'self' fonts.googleapis.com 'unsafe-inline';` +
       `font-src fonts.gstatic.com;` +
-      `connect-src 'self' blockscout-index.changi.dfi.team ${
+      `connect-src 'self' blockscout-index.changi.dfi.team solc-bin.ethereum.org raw.githubusercontent.com ${
```

Comment:
Since we aren't gonna be needing any other api calls from these 2 sites, is it possible to use the specific url here?

Reply:
Will add that specific path